docs/source/reference/ml.rst

```diff
@@ -8,7 +8,7 @@ Machine Learning utilities
 MLflow
 ------
 
-Arbitrary MLflow models can be used with Koalas Dataframes,
+Arbitrary MLflow models can be used with Koalas DataFrames,
```

Comment:
@dmatrix this typo should be fixed though. feel free to open a PR for this separately if you can find some time.